resources/assets/js/features/tripsList/actions.js

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios'
+import * as actions from './actionTypes';
+
+const url = '/api/v1/trips/';
+
+export function filterPast() {
+    return dispatch => {
+        axios.get(url+'?filter=past')
+            .then(response => {
+                dispatch({
+                    type: actions.TRIPSLIST_FILTER_PAST,
+                    filtered: response.data
+                })
+            })
+            .catch(error => {
+            });
+    };
+}
+
+export function filterUpcoming() {
```

Comment:
actions should reflects logic that they performs in this case it's FETCH_UPCOMING